funnel/templates/project.html.jinja2

```diff
@@ -17,6 +18,8 @@
       "name": {{ project.title|tojson }},
       "url": {{ project.url_for(_external=true)|tojson }},
       "startDate": {{ project.schedule_start_at_localized|tojson }},
+      "endDate": {{ project.schedule_end_at_localized|tojson }},
+      "eventStatus": "https://schema.org/EventScheduled",
```

Comment:
I think this needs an `if` condition that checks for `schedule_start_at` being null.

Reply:
We are checking it on line 13